src/index.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import './index.css';
 
 const container = document.getElementById('root');
 const root = createRoot(container);
 
 root.render(
-
-      <App />
```

Comment:
I would like to mention the opening "App" should be like this one <App> , it's the vs code autocorrection 😉